README.org

```diff
@@ -458,7 +458,9 @@ The ZK-Index buffer is in a major mode with a dedicated keymap:
     (define-key map (kbd "o") #'other-window)
     (define-key map (kbd "f") #'zk-index-focus)
     (define-key map (kbd "s") #'zk-index-search)
-    (define-key map (kbd "D") #'zk-index-switch-to-desktop)
+    (define-key map (kbd "g") #'zk-index-query-refresh)
+    (define-key map (kbd "d") #'zk-desktop-send-to-desktop)
+    (define-key map (kbd "D") #'zk-desktop-switch-to-desktop)
```

Comment:
Good to add query-refresh here.

But there shouldn't be zk-desktop functions here, since zk-index can be installed independent of zk-desktop.

Instead, these functions should be added to the zk-index-map in the configuration for zk-desktop.